standard/basic.php

```diff
@@ -53,6 +53,9 @@ function cli_set_process_title(string $title): bool {}
 
 /**
  * Returns the current process title
+ *
+ * Returns the current process title, as set by cli_set_process_title(). Note that this may not exactly match what is shown in ps or top, depending on your operating system.
```

Comment:
It's probably best to merge this line with the line above. Currently, we're getting a duplicated info which doesn't look nice in a quick doc:
<img width="337" alt="1" src="https://user-images.githubusercontent.com/33625946/162958170-3b1474a2-e257-4686-a1ae-9147c63e0366.png">